CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # avro-patches
 
+## v0.3.0
```

Comment:
It could, but I tend to only use those for bug fixes, and don't generally put improved performance into that category.